src/lib/parser.rs

```diff
@@ -128,12 +128,9 @@ impl<TokId: TryFrom<usize>> LexParser<TokId> {
         if orig_name == ";" {
             name = None;
         } else {
-            debug_assert!(orig_name.len() > 0);
-            if !((orig_name.chars().nth(0).unwrap() == '\''
-                  && orig_name.chars().last().unwrap() == '\'')
-                 ||
-                 (orig_name.chars().nth(0).unwrap() == '\"'
-                  && orig_name.chars().last().unwrap() == '"')) {
+            debug_assert!(!orig_name.is_empty());
+            if !(   (orig_name.starts_with('\'') && orig_name.ends_with('\''))
```

Comment:
Why the spaces between the parens?

Reply:
So the `||` lines up nicely.

Reply:
OK, first time I've seen something like this, but I'll believe you!